frontend/src/container/NewWidget/index.tsx

```diff
@@ -161,28 +163,38 @@ function NewWidget({ selectedGraph, saveSettingOfPanel }: Props): JSX.Element {
 		FeatureKeys.QUERY_BUILDER_PANELS,
 	);
 
+	const isSaveDisabled = useMemo(
+		() =>
+			isQueryBuilderActive &&
+			currentQuery.queryType === EQueryType.QUERY_BUILDER &&
+			currentQuery.builder.queryData.find(
+				(query) => query.dataSource !== DataSource.METRICS,
+			) !== undefined,
+		[
+			currentQuery.builder.queryData,
+			currentQuery.queryType,
+			isQueryBuilderActive,
+		],
+	);
+
 	return (
 		<Container>
 			<ButtonContainer>
-				{isQueryBuilderActive && (
```

Comment:
Since you are removing the `isQueryBuilderActive` from here, will it prevent users from creating the more panels for traces and logs?

Reply:
isQueryBuilderActive is powered from feature flag itself not other params